### ISSUE TITLE
Missing Ores 0.4.1

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses#16]
I have my mining hill and I have gone to level 6 and I am only getting gems, junk ore and copper ore. Shouldn't lead ore and other ores be available?

**Comment by teagan75** [comment]
Lead, tin and nickel will now be a second product from crushing ore. More reliable

**Comment by Ladysarajane** [comment]
So I need the crusher to crush junk ore to get lead, tin and nickel?

**Comment by teagan75** [comment]
They come off of different ore. And yes you will need the crusher.

**Comment by Ladysarajane** [comment]
Thanks...I am assuming the ore will be lower in the mine?

**Comment by teagan75** [comment]
iirc tin comes off copper, lead off silver and nickel off gold. There is another one that comes off Mystral but that’s for much later

**Comment by Ladysarajane** [comment]
Okay thanks. I have to wait till I get further to check that.

**Comment by Ladysarajane** [comment]
I am in the hamlet quest. I need to get tin to complete the bronze quest. I have been down to level 15 in the mines and I haven't found any new ores and isn't the crusher in the City Quest? How do I get tin to complete this quest?

**Comment by teagan75** [comment]
Ok. I knew I forgot about something 😩 I will have to add the ores to Brandon’s shop. The other mod was proving unreliable which is too bad. Will try and update tomorrow. I am ill today

**Comment by teagan75** [comment]
You are sure the crusher is not in the Hamlet technology line?

**Comment by Ladysarajane** [comment]
Hamlet doesn't have a technology line except for making the iron furnace and hoppers...it has a magic line and that is where I need to make bronze for the First Spell. 
![2019-07-01_17 37 36](https://user-images.githubusercontent.com/34258851/60468291-292d4100-9c27-11e9-9f0c-05e063d91aa7.png)

**Comment by Ladysarajane** [comment]
Whoa, I missed the post about you putting the ores in Brandon's shop. I hope you feel better soon. The crusher is not in the Hamlet Quest line as seen above in my image.In my last world the crusher showed in the city quest line.
I had a "pig invasion" in my town lol! I made the spawn eggs and triggered the commands and when I walked out of my house, my town had 10 pigs in it. My biome is forest. So the commands work well.
